src/components/com_tjdashboard/site/assets/js/tjDashboardUI.js

```diff
@@ -37,11 +38,15 @@ var TJDashboardUI = {
 					value.params = JSON.parse(value.params);
 				}
 
-				if(value.params.color!=undefined && value.params.color.length!=0){
+				if(value.params.color){
```

Comment:
Are you sure the params will always contain color ? You should change this to

`if (typeof(value.params.color) != 'undefined' && value.params.color)`

Reply:
Are you sure the params will always contain color ? You should change this to: 
Default color value is set before if